src/utils/metadata.ts

```diff
@@ -176,3 +177,21 @@ export function fetchIdpsMetadata(
       return mapIpdMetadata(idpMetadata, idpIds);
     });
 }
+
+export function parseStartupSpidStrategy(
```

Comment:
I'd add some comments on the format expected

Reply:
Fixed in a4bdb13